webapp/packages/core-authentication/src/UserInfoResource.ts

```diff
@@ -55,6 +55,10 @@ export class UserInfoResource extends CachedDataResource<UserInfo | null, void,
     return this.data !== null;
   }
 
+  get isAnonymous() {
+    return !this.data || this.data.isAnonymous;
+  }
```

Comment:
If we don't have this.data we don't know and can't say that the user is anonymous

better to use `this.data && this.data.isAnonymous`